file-adapter/src/main/java/org/polypheny/db/adapter/file/Condition.java

```diff
@@ -175,6 +175,41 @@ Object getParamValue( final DataContext dataContext, final PolyType polyType ) {
         return out;
     }
 
+    /**
+     * Implement the like keyword
+     *
+     * @param str Data in database
+     * @param expr String in SQL statement
+     * @return
+     */
+    private static boolean like( final String str, String expr ) {
+        final String[] parts = expr.split( "%" );
+        final boolean traillingOp = expr.endsWith( "%" );
+        expr = "";
+        for ( int i = 0, l = parts.length; i < l; ++i ) {
```

Comment:
@erdengk Please refrain from using single character variables like `l` that are not initialized in control structures (e.g. `for-loops`). This will be fairly hard to read for others.
Furthermore you only use `l` one other time so it is absolutely reasonable to just stick with `parts.length`.